lib/tasks_private/openstack.rake

```diff
@@ -4,82 +4,113 @@ namespace :vcr do
     task :delete => :environment do
       vcr_dir = ManageIQ::Providers::Openstack::Engine.root.join("spec/vcr_cassettes/manageiq/providers/openstack/cloud_manager")
       Dir.glob(vcr_dir.join("refresher*.yml")).each { |f| File.delete(f) }
+      Dir.glob(vcr_dir.join("refresher/*.yml")).each { |f| File.delete(f) }
     end
```

Comment:
Think we can do this in one line with `**/refresher*.yml` but this is good for now